apps/web/playwright/out-of-office.e2e.ts

```diff
@@ -25,14 +25,15 @@ test.describe("Out of office", () => {
 
     await page.getByTestId("notes_input").click();
     await page.getByTestId("notes_input").fill("Demo notes");
-    await page.getByTestId("create-entry-ooo-redirect").click();
+    await page.getByTestId("create-or-edit-entry-ooo-redirect").click();
 
     await expect(page.locator(`data-testid=table-redirect-n-a`)).toBeVisible();
   });
 
-  test.skip("User can configure booking redirect", async ({ page, users }) => {
+  test("User can configure and edit booking redirect", async ({ page, users }) => {
```

Comment:
Nah, this is wrong. Add a separate test for editing, similar to how there is a test for creating. For example, you can add a test called `User can edit out of office entry,` just like the test for `User can create out of office entry`

Reply:
@anikdhabal Done! Separated out the test into a different one.

Heads up - I am new to Playwright, so might make some mistakes. Apologies in advance. 😬